service-front/module/Application/view/application/pages/national_insurance_number.twig

```diff
@@ -0,0 +1,58 @@
+<div class="govuk-width-container">
+    <main class="govuk-main-wrapper">
+        <h1 class="govuk-heading-xl">National Insurance Number</h1>
+        <form method="POST" target="">
+            <h2 class="govuk-heading-m">Name</h2>
+            <p class="govuk-body">
+                {{ details_data.Name }}
+            </p>
+            <h2 class="govuk-heading-m">Date of Birth</h2>
+            <p class="govuk-body">
+                {{ details_data.DOB }}
+            </p>
+            <h2 class="govuk-heading-m">National Insurance Number</h2>
+               <p class="govuk-body">
+               </p>
+
+            <div class="govuk-form-group">
+                <h1 class="govuk-label-wrapper">
+                    <label class="govuk-label govuk-label--xs" for="event-name">
+                        A NINO is made up of 2 letters, 6 numbers and a final letter, which is always A, B, C or D. Example QQ
+                        12 34
+                        56 A
+                    </label>
+                </h1>
+                {% for message in form.get('nino').messages %}
+                    <p id="event-name-error" class="govuk-error-message">
+                        <span class="govuk-visually-hidden">Error:</span> {{ message }}
+                    </p>
+                {% endfor %}
+                <input class="govuk-input govuk-!-width-one-third" id="nino" name="nino" type="text" autocomplete="nino"
+                       value="{{ form.get('nino').value }}"/>
+            </div>
```

Comment:
A few fixes here:
- The "National Insurance Number" text should be inside the label
- Using h2 rather than h1
- Linking elements with aria-describedby
- Adding all the CSS classes when in an error state

```suggestion

            <div class="govuk-form-group {% if form.get('nino').messages %}govuk-form-group--error{% endif %}">
                <h2 class="govuk-label-wrapper">
                    <label class="govuk-label govuk-label--m" for="nino">
                        National Insurance Number
                    </label>
                </h2>

                <p id="nino-hint" class="govuk-hint">
                    A NINO is made up of 2 letters, 6 numbers and a final letter, which is always A, B, C or D. Example QQ 12 34 56 A
                </p>

                {% if form.get('nino').messages %}
                    <p id="nino-error" class="govuk-error-message">
                        <span class="govuk-visually-hidden">Error:</span>
                        {{ form.get('nino').messages | join(', ', ' and ') }}
                    </p>
                {% endif %}

                <input
                    class="govuk-input govuk-!-width-one-third {% if form.get('nino').messages %}govuk-input--error{% endif %}"
                    id="nino" name="nino" type="text" autocomplete="nino"
                    value="{{ form.get('nino').value }}" aria-describedby="nino-hint {% if form.get('nino').messages %}nino-error{% endif %}"
                />
            </div>
```